docs/proposal-rose-suite-run.md

```diff
@@ -24,26 +22,81 @@ The logical step is therefore to move these extra functionalities provided by
 Rose into Cylc - and retire `rose suite-run` and related utilities - so that
 all suites will start up with only Cylc commands in the future.
 
+## Work Plan
+
+- [ ] Agree future form of `cylc-flow.rc` [Cylc-Admin #43](https://github.com/cylc/cylc-admin/issues/43)
+  - [x] Create new issue against cylc-admin
+  - [ ] Create a template `flow.rc`, listing all the options
+    to be available in the combined `flow.rc`. [Link to it](example_files/flow.rc)
+  - [ ] Deprecate settings for job host?
+  - [ ] Add sections for `[job platforms]`
+
+
+- [ ] Implement new `cylc-flow.rc` form. [Cylc-flow #3260](https://github.com/cylc/cylc-flow/issues/3260)
+  - [ ] Check old tests for `global.rc` & `suite.rc` to ensure that functionality
+    is not lost.
+  - [ ] Devise tests for the new flow.rc
+  - [ ] Create new config parser, perhaps called `cylc-flow-rc-parser.py`
```

Comment:
Wait what! You do not want to re-write the parsec config parser.

Reply:
Quite.